commit e5b3b9156b7e9f4f1d02bc98c116b47d02e4f607 (refs/changes/28/4728/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2018-08-20T18:07:48-07:00 (1 year, 2 months ago)
    
    TRITON-710 vm-agent needs new restify-clients to fix BadDigest errors with new node

diff --git a/lib/vmapi-client.js b/lib/vmapi-client.js
index 44ff068..86302b5 100644
--- a/lib/vmapi-client.js
+++ b/lib/vmapi-client.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2015, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -135,7 +135,10 @@ function VmapiClient(options) {
     this.client = restify.createJsonClient({
         url: options.url,
         log: options.log,
-        userAgent: options.userAgent
+        userAgent: options.userAgent,
+        contentMd5: {
+            encodings: ['utf8', 'binary']
+        }
     });
 }
 
diff --git a/package.json b/package.json
index cc9fcad..94929fe 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "vm-agent",
     "description": "SmartDC Node VM Agent",
-    "version": "1.7.1",
+    "version": "1.7.2",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
@@ -15,7 +15,7 @@
         "lstream": "0.0.4",
         "mockery": "1.4.0",
         "node-uuid": "1.4.7",
-        "restify-clients": "1.1.1",
+        "restify-clients": "1.6.0",
         "tape": "4.2.2",
         "vasync": "1.6.3",
         "vmadm": "1.1.0"
